ci/testdata/Dockerfile

```diff
@@ -3,6 +3,7 @@ FROM golang:1.19-alpine
 # Install bash and git (and build-base to get gcc)
 # (this is required when building FROM: golang:alpine)
 RUN apk update && apk add --no-cache git bash build-base
```

Comment:
This line seems to work:
```docker
RUN apk update && apk add --no-cache git=~2.47 bash=~5.2.37 build-base=~0.5
```
To experiment with different versions:
```sh
docker pull alpine
docker run -it --rm alpine ash
```
The when running interactively in the docker container, use the `apk` commands, like:
```sh
apk list
apk update
apk add --no-cache git bash build-base
apk update && apk add --no-cache git=~2.47 bash=~5.2.37 build-base=~0.5
```
etc.